### PR TITLE
Post `osbuild/images` split autumn cleanup :broom:

### PR DESCRIPTION
**Pull request by thozza**
I started working on the mountpoint policy change and realized, that the `pathpolicy` code was still present in both, osbuild-composer and images repository. However, the copy present in osbuild-composer repository was not used any more. This made me have a look at some other internal packages, which led to deleting some additional unused copies of code.

There is still the `internal/fsnode` package, which should be removed (and the code in BP customizations which uses it), but this would require reworking the files / dir customization validation.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by supakeen** [comment]
Awesome; red diffs are my favorite.

**Review by supakeen — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -195,6 +196,10 @@ worker-key-pair: ca
 	# delete the request
 	rm /etc/osbuild-composer/worker-csr.pem
 
+.PHONY: unit-tests
+unit-tests:
+	go test -race ./...
+	go test -race ./internal/dnfjson/... -force-dnf
```
Comment (achilleas-k) [inline Makefile:202]: This is really nice :heart:

**Review by achilleas-k — APPROVED** [review]
Amazing.  Thank you!!!

File: Makefile (hunk 1 above)
Comment (thozza) [reply Makefile:202]: :bowing_man:

**Pull request merged**